src/cms/index.js

```diff
@@ -162,6 +178,28 @@ export function renderBlocks({
 }) {
   return (
     <>
+      {Array.from(
+        document.querySelectorAll(`div[id^=${TYPES.INDICATOR_WIDGET}]`)
+      ).map(el => {
+        // deprecated php indicator used to have innerhtml in its container div
+        // eslint-disable-next-line no-param-reassign
+        el.innerHTML = '';
+        return ReactDOM.createPortal(
+          <Card
+            id={el.getAttribute('id')}
```

Comment:
Should this not be `POST_ID` ? If not still make it a new const.

Reply:
Made it a constant. Indicators are not custom post type, so we can't have `POST_ID`